focus_areas/community_growth.md

```diff
@@ -5,17 +5,17 @@ Goal: Identify the size of the project community and whether it's growing, shrin
 Metric | Quesiton
 --- | ---
 [Contributors](../metrics/contributors.md) | What is the number of contributors?
-[New Overall Contributors](../metrics/contributors-new.md) | What is the overall number of new contributors?
+[New Contributors](../metrics/contributors-new.md) | What is the overall number of new contributors?
 [New Contributors of Commits](../metrics/pull-requests-merge-contributor-new.md) | What is the number of persons contributing with an accepted commit for the first time?
-[New Contributors of Opened Issues](../metrics/issues-first-time-opened.md) | What is the number of persons opening an issue for the first time?
-[New Contributors of Closed Issues](../metrics/issues-first-time-closed.md) | What is the number of persons closing an issue for the first time?
-[New Contributors of Initiated Code Reviews](../metrics/pull-requests-initiated-reviews-contributors-new.md) | What is the number of persons initiating a code review for the first time?
-[New Contributors of Reviews for Code](../metrics/pull-requests-code-reviews-contributors-new.md) | What is the number of persons contributing with reviews of code for the first time?
-[New Contributors of Posted Messages](../metrics/mailing-lists-messages-contributors-new.md) | What is the number of persons posting messages in mailing lists for the first time?
+[Open Issues New Contributors](../metrics/issues-first-time-opened.md) | What is the number of persons opening an issue for the first time?
```

Comment:
Minor comment: "opened" tried to emphasize the fact that this metric refers to issues that are opened during a certain period by new contributors, as opossed to issues that are still open during a certain period, and that were opened by new contributors.

For example, during January, in a certain project 15 new people could be found opening bug reports for the first time. But during the same time, maybe there are 18 people who submitted bug reports that are still open (some of them were opened in November, December, and are still open, while some of those opened during January, maybe were closed during that same month).

Reply:
Lets discuss at next GMD meeting.